layouts/page/pulumi-up.html

```diff
@@ -0,0 +1,34 @@
+{{ define "main" }}
+    <div class="pulumi-up-body text-white py-12">
+        <div class="container mx-auto">
+            <section id="about">
+                <div class="neon-container">
+                    <span class="blue-neon-sign-xl">Pulumi</span>
+                    <span class="green-neon-sign-xl">UP</span>
+                </div>
+                <pulumi-date-countdown
+                    value-label-class="text-2xl"
+                    text-class="w-full text-center text-white text-5xl date-countdown"
+                    date-string="2021-04-20T09:00:00-08:00"
```

Comment:
Seems like this should go into the page metadata (frontmatter). What do you think?